### ISSUE TITLE
[English]: Operator

### ISSUE THREAD

**Issue by JasonMorgan** [cncf/glossary#445]
### Details (optional)

Define what an operator is. Will also require a definition of operator pattern. 

This issue was inspired by [this article](https://thenewstack.io/hello-kubernetes-operators-and-vendors-few-people-know-what-youre-selling/) which covered some terms that seem to be somewhat ambiguous.

**Comment by gullapalli9482** [comment]
If the issue is not taken by anyone, I would like to undertake it.

**Comment by CathPag** [comment]
Looks like you have still a PR open for [technical debt](https://github.com/cncf/glossary/pull/500). Once that is complete, we can assign ou a new term

**Comment by gullapalli9482** [comment]
Oh, I forgot to mention that 🥲 before asking for the term.

**Comment by CathPag** [comment]
No worries. Check back in once the other term is done :)

**Comment by gullapalli9482** [comment]
How do i reach out to the maintainers for reviewing the pull request , apart from the conversation done here, could I do it in slack, is it correct , could I do it?

**Comment by CathPag** [comment]
It is on our radar. We are all doing this in our spare time and ask for patience when waiting for feedback. Contributing to open source doesn't stop with submitting a PR. Please wait to hear from us for next steps.

**Comment by shambashib20** [comment]
Can I work on this issue? I have already worked on a pr on Idempotence!

**Comment by CathPag** [comment]
There you go, @shambashib20 :)

**Comment by CathPag** [comment]
Actually, hold on @shambashib20. We are still evaluating if this should be part of the glossary.

**Comment by shambashib20** [comment]
Is this issue open or I have start to find new issue?

**Comment by gullapalli9482** [comment]
It means the decision to have the "operator" term under the glossary is still pending.

**Comment by shambashib20** [comment]
Oh okay

**Comment by CathPag** [comment]
We are currently discussing the boundaries of the glossary and this discussion may have to wait until our next maintainer meeting in a few weeks. It's not easy to establish, so we'll need a little more time. Thanks for your patience!

**Comment by CathPag** [comment]
This term is too close to operator pattern (#446) and one term is enough for the glossary.
